### ISSUE TITLE
DNS problem 

### ISSUE THREAD

**Issue by MrAurel** [haugene/docker-transmission-openvpn#2836]
### Is there a pinned issue for this?

- [X] I have read the pinned issues and could not find my issue

### Is there an existing or similar issue/discussion for this?

- [X] I have searched the existing issues
- [X] I have searched the existing discussions

### Is there any comment in the documentation for this?

- [X] I have read the documentation, especially the FAQ and Troubleshooting parts

### Is this related to a provider?

- [X] I have checked the provider repo for issues
- [X] My issue is NOT related to a provider

### Are you using the latest release?

- [X] I am using the latest release

### Have you tried using the dev branch latest?

- [ ] I have tried using dev branch

### Docker run config used

version: '3.3'
services:
    transmission-openvpn:
        cap_add:
            - NET_ADMIN
        volumes:
            - '/volume1/docker/transmission-openvpn/data/:/data'
            - '/volume1/docker/transmission-openvpn/config/path/:/config'
        environment:
            - OPENVPN_PROVIDER=NORDVPN
            - OPENVPN_CONFIG=france
            - OPENVPN_USERNAME=*********
            - OPENVPN_PASSWORD=********
            - LOCAL_NETWORK=192.168.1.0/24
            - NORDVPN_COUNTRY=FR
        logging:
            driver: json-file
            options:
                max-size: 10m
        ports:
            - '9091:9091'
        image: haugene/transmission-openvpn
        container_name: transmission-vpn
        dns:
            - '8.8.8.8'
            - '8.8.4.4'
        security_opt:
            - seccomp:unconfined

### Current Behavior

Cannot start

### Expected Behavior

Start

### How have you tried to solve the problem?

I see the pinned post #2410 but it cannot resolve my problem

### Log output

Starting container with revision: 07f5a2b9aea5028c9bb75438c1552708e91dde71
TRANSMISSION_HOME is currently set to: /config/transmission-home
/etc/openvpn/start.sh: line 51:    10 Aborted                 (core dumped) nslookup ${HEALTH_CHECK_HOST:-"google.com"} > /dev/null 2>&1
WARNING: initial DNS resolution test failed
Creating TUN device /dev/net/tun
Using OpenVPN provider: NORDVPN
Running with VPN_CONFIG_SOURCE auto
Provider NORDVPN has a bundled setup script. Defaulting to internal config
Executing setup script for NORDVPN
/etc/openvpn/nordvpn/..

### HW/SW Environment

```markdown
- OS:DSM 7
- Docker:26.1.1
```

**Comment by pkishino** [comment]
Any reason you didn’t post the whole log?

**Comment by jrlambs** [comment]
I had this same issue (specifically with privado, and the exact same url).  that appears to be the default config, but they have a bunch of other servers.  I went to privado's site and had it generate an ovpn file for me.. when I opened it up it gave me a different url to use.  specifically atl-007.vpn.privado.io (but I think it was based on a location I selected).  anyway... I then found this documentation about setting the OPENVPN_CONFIG env variable.  you can use the variable to select which available config for your provider you want to use.

basically I just added this line to my docker-compose file under environment variables and now it works.

OPENVPN_CONFIG=atl-007

Documentation here:  https://haugene.github.io/docker-transmission-openvpn/run-container/ 

The documentation walks you through how you can figure out what options are available to put in that spot for your vpn provider. Look at #2 under "Things to Remember"

**Comment by jhaury** [comment]
> ... I went to privado's site and had it generate an ovpn file for me.. when I opened it up it gave me a different url to use. specifically atl-007.vpn.privado.io (but I think it was based on a location I selected). anyway... I then found this documentation about setting the OPENVPN_CONFIG env variable. you can use the variable to select which available config for your provider you want to use.
> 
> basically I just added this line to my docker-compose file under environment variables and now it works.
> 
> OPENVPN_CONFIG=atl-007

You are my hero! My transmission has been down for a minute and this was the fix. I did the same thing, I went to Privado's website and logged in and created an ovpn file and extracted the server from the file. Mine was dfw-062 (Dallas).

Many thanks!

**Comment by pilotman28** [comment]
There is a script which is provided with the image to help with this. The script is located at, 'config/vpn-configs-contrib/openvpn/privado/updateConfigs.sh'. The only customization necessary here is to change which '.opvn' file is symlinked to 'default.opvn' to be whatever you wish. I'll submit a code change to the shell script to take an argument if given to make it easily changed but in the mean time the shell script can be modified directly to be to your liking, 'ln -s atl-007.opvn default.ovpn' in my case.
